FabLabKasse/gui.py

```diff
@@ -234,14 +234,22 @@ def __init__(self):
         self.startupProgress.show()
 
         self.cashPollTimer = QtCore.QTimer()
-        self.cashPollTimer.setInterval(500) # start with fast interval, later reduced
+        self.cashPollTimer.setInterval(500)  # start with fast interval, later reduced
         self.cashPollTimer.timeout.connect(self.pollCashDevices)
         self.cashPollTimer.start()
 
-        self.pushButton_load_cart_from_app.setVisible(cfg.has_option("mobile_app","enabled") and cfg.getboolean("mobile_app","enabled"))
+        self.idleCheckTimer = QtCore.QTimer()
+        self.idleCheckTimer.setInterval(10000)
+        self.idleCheckTimer.timeout.connect(self._reset_if_idle)
+        self.idleCheckTimer.start()
+
+        self.CATEGORY_VIEW_RESET_TIME = 1800000  # idle threshold time in ms, is half an hour
```

Comment:
as it is only used here (and referenced in a comment somewhere else), we can make this a local variable, or even better fetch the time in seconds via config.get(...,default=1800)